src/main/components/uml-element/assessable/assessable.tsx

```diff
@@ -151,10 +151,13 @@ export const assessable = (
           console.error('Could not parse artemis sgi', e);
           return;
         }
+        // TODO: following messages should be received from Artemis
+        const removeMessage = 'Do you want to remove the link to the assessment instruction?';
+        const tooltipMessage = 'Assessment Instruction: ' + instruction.instructionDescription;
         const { id: elementId, assessment } = this.props;
         const score = instruction.credits;
         const feedback = instruction.feedback;
-        const dropInfo = { instruction };
+        const dropInfo = { instruction, removeMessage, tooltipMessage };
```

Comment:
this could be typed with an Interface to make it easier to work with. 
```
interface DropInformation {
    instruction: string,
    removeMessage: string,
    tooltipMessage: string,
}
```
or as type  such as `type DispatchProps`

Reply:
actually we defined `dropInfo ` as any in here:https://github.com/ls1intum/Apollon/blob/069e448d6fe14ae1914f9f56927f5284dcbd46f9/src/main/services/assessment/assessment.ts#L6

This way, dropInfo can be used for some other needs in later. So I did not specify interface in here at first.